pyproject.toml

```diff
@@ -100,6 +101,7 @@ dependencies = [
     "beautifulsoup4",
     "boto3",
     "botocore",
+    "dbt-postgres",
```

Comment:
this is needed because the KPDB build uses the `nycplanning/build-base` image and runs `python3 -m pip install . -c ./python/constraints.txt`. therefore this list of packages are the only ones installed

this wasn't needed to use dbt in Template DB beacuse it uses the `nycplanning/dev` image which has all packges listed in requirements.txt

Reply:
build-base has its own requirements file in `docker/build-base`, this spot is for dependencies for dcpy. This also should be added to the main requirements.in if it's not!